Source/inv.cpp

```diff
@@ -358,9 +358,9 @@ void CheckInvPaste(int pnum, Point cursorPosition)
 	auto &player = Players[pnum];
 
 	SetICursor(player.HoldItem._iCurs + CURSOR_FIRSTITEM);
-	int i = cursorPosition.x + (IsHardwareCursor() ? 0 : (icursW / 2));
-	int j = cursorPosition.y + (IsHardwareCursor() ? 0 : (icursH / 2));
-	Size itemSize { icursW28, icursH28 };
+	int i = cursorPosition.x + (IsHardwareCursor() ? 0 : (icursSize.width / 2));
+	int j = cursorPosition.y + (IsHardwareCursor() ? 0 : (icursSize.height / 2));

```

Comment:
This function reuses `i`/`j` as loop counters way later. These two assignments are really only used for the for loop on line 366.

Pointing this out not to say it needs to change in this PR, but to highlight the mess as a reason to leave it for a future PR 😄